mula/Makefile

```diff
@@ -67,6 +67,10 @@ cov: ## Generate a test coverage report
 		--cov-report term-missing:skip-covered \
 		--cov=scheduler tests/
 
+shell: ## Start a shell in the scheduler container.
```

Comment:
The comment is a bit misleading, since this actually starts a *new* container with a Python shell as the entry point, and not exec into a running container (at least this is how I read it the first time)

Reply:
Yeah I'll remove it since, I'm not using it at the moment. And indeed the comment is a bit misleading